posts/ilaborde/1.automapper.md

```diff
@@ -0,0 +1,170 @@
+---
```

Comment:
Rename file to `01.automapper.md` to have a good order when you have more than 9 posts in the future.